vit-servicing-station-tests/src/common/data/generator/voting/plan.rs

```diff
@@ -155,6 +161,10 @@ impl ValidVotePlanGenerator {
             .parameters
             .registration_snapshot_time
             .unwrap_or(voting_start);
```

Comment:
I am wondering, does `voting_start` variable is the same as `voting_tally_start`  ?

Reply:
No, voting start is the date in where ballots can be submitted.  Voting tally is the date in where the tally calculation starts.